app/src/components/dataset/DatasetForm.tsx

```diff
@@ -125,7 +127,9 @@ export function DatasetForm({
       >
         <Flex direction="row" justifyContent="end">
           <Button
-            disabled={!isDirty}
+            // Only allow submission if the form is dirty for edits
+            // When creating allow the user to click create without any changes as the form will be prefilled with valid values
+            disabled={formMode === "edit" ? !isDirty : false}
```

Comment:
this was something i noticed, this !isDirty made it so a user had to make edits in the create flow - was originally added so that updates couldn't just be spam clicked